packages/@webex/internal-plugin-voicea/src/voicea.ts

```diff
@@ -388,6 +407,27 @@ export class VoiceaChannel extends WebexPlugin implements IVoiceaChannel {
     });
   };
 
+  /**
+   * Toggle turn on manual caption
+   * @param {bool} enable if true manual caption is turned on
+   * @returns {Promise}
+   */
+  public toggleManualCaption = (enable: boolean): undefined | Promise<void> => {
+    // @ts-ignore
+    return this.request({
+      method: 'PUT',
+      // @ts-ignore
+      url: `${this.webex.internal.llm.getLocusUrl()}/controls/`,
+      body: {
+        manualCaption: {
+          enable,
+        },
+      },
+    }).then((): undefined | Promise<void> => {
+      return undefined;
```

Comment:
suggestion: Not sure you really need this `.then`

Reply:
Just keep the function here in case we will implement host functionality for manual caption feature